symusic/factory.py

```diff
@@ -60,7 +61,7 @@ def __call__(self, ttype: Union[smt.TimeUnit, str]) -> smt.TimeUnit:
             ttype.is_time_unit()
             return ttype
         except AttributeError:
-            raise TypeError(f"{ttype} is not a TimeUnit object")
+            raise TypeError(f"{ttype} is not a TimeUnit object") from None
```

Comment:
What's the `from None` for?

Reply:
To indicate what's causing the issue, flagged by ruff
I actually just updated it with `except AttributeError as e` and `from e`